tests/conftest.py

```diff
@@ -41,9 +41,18 @@ def before_record_response(response):
     return before_record_response
 
 
+def get_record_mode():
+    record = os.getenv("RECORD", "none")
+    if record.lower() == "true":
+        return "all"
+    else:
+        return "none"
```

Comment:
There are three cases `RECORD=true` for recording, `RECORD=false` for replaying and `RECORD=none` for using the real API without recording

Reply:
I don't think there's an option to not record : https://vcrpy.readthedocs.io/en/latest/usage.html#record-modes
Unless you pass the --disable-recording flag

Reply:
We have these 3 modes in the API clients, so let's have them here too